lib/svg-sprite/mode/standalone.js

```diff
@@ -89,13 +89,22 @@ SVGSpriteStandalone.prototype._layout = function(files, cb, extend) {
                 }
             };
 
+            // Create the SVG setter
             Object.defineProperty(this.data.shapes[index], '_svg', {
                 enumerable: false,
                 writable: true
             });
 
-            this.data.shapes[index].__defineSetter__('svg', function(svg) {
-                this._svg = svg;
+            // Create the SVG getter/setter
+            Object.defineProperty(this.data.shapes[index], 'svg', {
+                enumerable: true,
```

Comment:
Need a double check for these 2 properties

Reply:
it better to either remove this props or set the default [values](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/defineProperty#description) (false)

Reply:
They were needed IIRC.

Reply:
No, they has default values if not provided. It's not critical here but it adds new functionality here. I'd rather go without it

Reply:
Like I said, they are needed:

```
test/svg-sprite/minimal-configuration/modes/defs.test.js (5.687 s)
  ● svg-sprite: default: «defs» mode › creates a visually correct stylesheet resource in CSS format

    TypeError: Cannot redefine property: svg
        at Function.defineProperty (<anonymous>)

      47 |
      48 |         // Create the SVG getter
    > 49 |         Object.defineProperty(dataShape, 'svg', {
         |                ^
      50 |             get() {
      51 |                 return this._svg || shape.getSVG(true, shapeDOM => {
      52 |                     shapeDOM.setAttribute('id', shape.id);
```

Reply:
Hmm, strange. But OK

Reply:
You can try dropping any of the two, maybe I missed something.

Reply:
@Kreeg ^^